charts/stable/paperless/templates/common.yaml

```diff
@@ -1 +1,11 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* Append the hardcoded settings */}}
+{{- define "paperless.harcodedValues" -}}
+env:
+    PAPERLESS_URL: {{ (first .Values.ingress.main.hosts).host }}
```

Comment:
Wouldn't doing it like this always override any custom values that have been set by the user?

Reply:
I did a merge instead of mergeOverride to avoid overriding user values

Reply:
.. but override is not doing what I was expecting...